api/requests.js

```diff
@@ -0,0 +1,31 @@
+const countriesUrl = 'http://localhost:3000/countries';
+
+export const getData = async () => {
+  try {
```

Comment:
Let's just return the response from the APi and use `try {} catch {}` when we call it